api_tests/nodes/filters/test_filters.py

```diff
@@ -94,3 +94,103 @@ def test_contributor_filter(self):
         res = self.app.get('{}{}'.format(self.contributors_url, self.user_two._id), auth=self.user.auth)
         actual = [node['id'] for node in res.json['data']]
         assert_equal(expected, actual)
+
+
+class NodesListDateFilteringMixin(object):
```

Comment:
Is this mixin necessary if there's only one subclass of it? We're trying to move away from these.